app/api/datastore/base.rb

```diff
@@ -12,6 +12,8 @@ class Base < Grape::API
     end
 
     rescue_from LaaCrimeSchemas::Errors::ValidationError do |ex|
+      Rails.error.report(ex, handled: true)
+
```

Comment:
Schema errors were not being reported. I think the client should probably be doing the reporting of these but I've added so we get flagged for the time being.